sycl/test-e2e/DeviceLib/math_windows_test.cpp

```diff
@@ -1,10 +1,15 @@
-// UNSUPPORTED: windows
-// Disabled on windows due to bug VS 2019 missing math builtins
+// REQUIRES: windows
 
-// REQUIRES: (accelerator || cpu) && windows
-// RUN: %{build} -c -o %t.o
-// RUN: %clangxx -fsycl %t.o %sycl_libs_dir/../bin/libsycl-cmath.o -o %t.out
+// TODO: Add hypotf case back when the missing symbol is fixed.
```

Comment:
Is there an issue for this? If not, should we open one?

Reply:
That is the plan indeed. There is another test affected by it, so I will bundle them together.